synapse/third_party_rules/access_rules.py

```diff
@@ -110,6 +110,10 @@ def on_create_room(
         If yes, make sure the event is correct. Otherwise, append an event with the
         default rule to the initial state.
 
+        Checks if a m.rooms.power_levels event is being set during room creation.
+        If yes, make sure the event is allowed. Otherwise, append an event with custom
+        default rules to the initial state.
```

Comment:
afaict we *don't* append such an event, but rather set the `power_level_content_override`?

Reply:
Oh! Yes indeed, sorry I misread your last review comment. This has been corrected :slightly_smiling_face: